lib/wanda_web/api_spec.ex

```diff
@@ -0,0 +1,24 @@
+defmodule WandaWeb.ApiSpec do
```

Comment:
In trento web we faced a similar situation in relation to apis/openapi/openapispex, see [here](https://github.com/trento-project/web/tree/main/lib/trento_web/openapi)

Let's consider If there's some valuable learning we can bring here - even on what not to do :sweat_smile: